Models/Tip.cs

```diff
@@ -4,59 +4,33 @@
 
 namespace SmartSaver.Models
 {
-    class Tip
+    public class Tip
     {
-        private string[] tipNames;
-        private string[] tipLinks;
 
-        public string[] TipNames
+        private string tipName;
+        private string tipLink;
+
+        public Tip()
         {
-            get
-            {
-                return tipNames;
-            }
-            set
-            {
-                tipNames = value;
-            }
         }
```

Comment:
Could be written in better fashion.
`public class Genre
{
    public string Name { get; set; }
}`